phpcs-ruleset.xml

```diff
@@ -82,6 +82,11 @@
         <exclude-pattern>profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tests/tmgmt_poetry.test</exclude-pattern>
     </rule>
 
+    <!-- Custom helper modules should be included in the feature folder for this case. -->
+    <rule ref="QualityAssurance.InfoFiles.Features.CustomModule">
+        <exclude-pattern>profiles/common/modules/features/nexteuropa_dgt_connector/**</exclude-pattern>
```

Comment:
I think you need to also exclude those from the profiles. It might not be necessary in jenkins anymore since we run cs before the build, but can be annoying locally